pycsghub/cli.py

```diff
@@ -0,0 +1,56 @@
+import typer
+from typing import Annotated, List, Optional
+from pycsghub.cmd import repo
+from pycsghub.cmd.repo_types import RepoType
+from pycsghub.constants import DEFAULT_CSGHUB_DOMAIN, DEFAULT_REVISION
+
+app = typer.Typer(add_completion=False)
+
+OPTIONS = {
+    "repoType": typer.Option(default=..., help="Specify the repository type."),
```

Comment:
default to `model`

Reply:
it's already set at method #22.

repo_type: Annotated[RepoType, OPTIONS["repoType"]] = RepoType.MODEL,

Reply:
> it's already set at method #22.
> 
> repo_type: Annotated[RepoType, OPTIONS["repoType"]] = RepoType.MODEL,

then, we can change the example to not input `--repo-type`